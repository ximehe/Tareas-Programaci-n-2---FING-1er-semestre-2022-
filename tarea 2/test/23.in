# implementar infoEnColCadenas
insertarEnColCadenas 3 3.0 0
insertarEnColCadenas 3 13.1 0
insertarEnColCadenas 3 13.1 9
insertarEnColCadenas 3 13.1 5
insertarEnColCadenas 8 8.0 0
insertarEnColCadenas 9 9.0 0
insertarEnColCadenas 5 5.0 5
insertarEnColCadenas 6 6.0 5
insertarEnColCadenas 7 7.0 5
insertarEnColCadenas 4 4.0 9
infoEnColCadenas 3 0
infoEnColCadenas 3 5
infoEnColCadenas 3 9
infoEnColCadenas 9 0
infoEnColCadenas 6 5
infoEnColCadenas 4 9
Fin