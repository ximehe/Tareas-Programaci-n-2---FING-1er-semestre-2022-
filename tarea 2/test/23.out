1>#  implementar infoEnColCadenas.
2>Se insertó (3,3.00) en la cadena 0.
3>Se insertó (3,13.10) en la cadena 0.
4>Se insertó (3,13.10) en la cadena 9.
5>Se insertó (3,13.10) en la cadena 5.
6>Se insertó (8,8.00) en la cadena 0.
7>Se insertó (9,9.00) en la cadena 0.
8>Se insertó (5,5.00) en la cadena 5.
9>Se insertó (6,6.00) en la cadena 5.
10>Se insertó (7,7.00) en la cadena 5.
11>Se insertó (4,4.00) en la cadena 9.
12>La primera aparición de 3 en la cadena 0 es (3,13.10)
13>La primera aparición de 3 en la cadena 5 es (3,13.10)
14>La primera aparición de 3 en la cadena 9 es (3,13.10)
15>La primera aparición de 9 en la cadena 0 es (9,9.00)
16>La primera aparición de 6 en la cadena 5 es (6,6.00)
17>La primera aparición de 4 en la cadena 9 es (4,4.00)
18>Fin.
